tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Package/PackageResponseBase.cs

```diff
@@ -38,5 +57,23 @@ public void SetPackageType(string packageType)
                 PackageType = type;
             }
         }
+
+        public static SdkLanguage GetLanguageFromRepo(string? repoName)
+        {
+            if (string.IsNullOrEmpty(repoName))
+            {
+                return SdkLanguage.Unknown;
+            }
+            repoName = repoName.ToLower();
+            return repoName switch
+            {
+                "azure-sdk-for-net" => SdkLanguage.DotNet,
```

Comment:
Would it make sense to use a constants map for this, since the repo names are fixed?

Reply:
+1. I see this map can be used in other places and can we move it to common constant class?